test/npcomp-run-mlir/invalid-input-shapes.mlir

```diff
@@ -0,0 +1,22 @@
+// RUN: not npcomp-run-mlir %s \
+// RUN:   -invoke invalid_input_shape \
+// RUN:   -arg-value="dense<1.0> : tensor<2x2x2x2xf32>" \
+// RUN:   -shared-libs=%npcomp_runtime_shlib 2>&1 \
+// RUN:   | FileCheck %s -check-prefix=ARG0-INVALID
+
+// RUN: not npcomp-run-mlir %s \
+// RUN:   -invoke invalid_input_shape_arg1 \
+// RUN:   -arg-value="dense<1.0> : tensor<1x2x5xf32>" \
+// RUN:   -arg-value="dense<1.0> : tensor<1x2x10xf32>" \
+// RUN:   -shared-libs=%npcomp_runtime_shlib 2>&1 \
+// RUN:   | FileCheck %s -check-prefix=ARG1-INVALID
+
+// ARG0-INVALID: invoking 'invalid_input_shape': input shape mismatch (%arg0).
```

Comment:
can you give more information?

Reply:
I updated the error messages to include the mismatching shapes. However, I had to `#include <sstream>` to do it in JITModule.cpp since to basically do a `StrJoin` on the shape for printing. The `invalid-input-shapes.mlir` test was also improved to check for the shape information in the error message.